src/proxy/server.ts

```diff
@@ -1,21 +1,24 @@
 import fs from "fs";
-import path from "path";
 import http from "http";
 import httpProxy from "http-proxy";
+import path from "path";
+import { DEFAULT_CONFIG } from "../config";
+import { decodeCookie, findSWAConfigFile, isHttpUrl, validateCookie } from "../core/utils";
+import { customRoutes, globalHeaders, mimeTypes, responseOverrides } from "./routes-engine/index";
+import { navigationFallback } from "./routes-engine/rules/navigationFallback";
+
 const proxyApp = httpProxy.createProxyServer({ autoRewrite: true });
 const proxyApi = httpProxy.createProxyServer({ autoRewrite: true });
 const proxyAuth = httpProxy.createProxyServer({ autoRewrite: false });
-import { decodeCookie, isHttpUrl, validateCookie } from "../core/utils";
-import { DEFAULT_CONFIG } from "../config";
 
-const buildAdress = (host: string, port: number | string | undefined) => `http://${host}:${port}`;
+const address = (host: string, port: number | string | undefined) => `http://${host}:${port}`;
```

Comment:
What would lead to `port` being `undefined`? This does run the risk of the output being `http://<host>:undefined`

Reply:
Same goes for `host`. These values are populated by the user's environment vars, and the default values are set by the CLI flags.

I am gonna add some checks but the user should still be able to override values using their own env vars.